recipes/hydra/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hydra" %}
+{% set version = "0.11.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 6d5b15297f486cf0a36fc7f5c8640004896ae3b878e8d236872fc0d74e823d4d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - omegaconf <=2.0
+    - typing-extensions
+
+test:
+  imports:
+    - hydra
+  requires:
+    - pytest
+  source_files:
+    - tests
+    - pytest.ini
+    - ATTRIBUTION
+    - examples
+    - hydra/test_utils
+  commands:
+    - pytest -k 'not conf_loader'  # conf_loader test doesn't find config.yml
```

Comment:
the conf loader is a critical part of Hydra. it not working is suggesting something is wrong.